helsingborg-stad/app-mitt-helsingborg
# Fix checkbox atom styling to match design

## Explain the changes you’ve made
Changes the styling of the checkbox atom to match design. Note, only the box itself, not the text and background. See screenshot of design: 
![Screenshot 2020-11-06 at 08 31 41](https://user-images.githubusercontent.com/7421890/98338714-e5143280-200a-11eb-926f-37ebce94cd72.png)


## Explain your solution

Adjusted which colors was used to use the new theme and its primary and complementary colors. 
Adjusted the size and margins to get closer to the design. 

Also added  a missing green primary color to the theme, and updated the colors and texts in the story.

## How to test the changes?

Concrete example:
1. Checkout this branch
2. Swap the application to run storybook
3. Fire upp the simulator by running the command`yarn ios`
4. Go into storybook and check the story for atoms. Or go into any form with checkboxes.  

## Was this feature tested in the following environments?
- [] Building the Application on a Android device/simulator.
